backend/src/Data/DbSeeder.cs

```diff
@@ -143,6 +143,10 @@ ILogger<DbSeeder> logger
                 {
                     await CreateUserAsync(manager, userInfo, appSettings.BootstrapUserPassword, logger).ConfigureAwait(false);
                 }
+            if (await manager.FindByEmailAsync("admin@admin.de").ConfigureAwait(false) is null)
+            {
+                await CreateUserAsync(manager, ("admin", "admin@admin.de", Enumerations.UserRole.ADMINISTRATOR), "Admin2005!", logger).ConfigureAwait(false);
```

Comment:
The `DbSeeder` already creates a bootstrap administrator user with a password from the `.env` file, see https://github.com/building-envelope-data/metabase/blob/55aa73cc85e4e849e729435faf803a03f4c7b498/backend/src/Data/DbSeeder.cs#L144